plugins/headless/hu.bme.mit.gamma.headless.server/resources/config.properties

```diff
@@ -3,3 +3,5 @@ headless.gamma.path=D:/2-Programming/Work/BME2021/MUNKA/openapi_linux_theta/ecli
 
 %Desired location of the workspaces
 root.of.workspaces.path=D:/2-Programming/Work/BME2021/MUNKA/theta_workspaces/
+
+logger.output.directory=D:/2-Programming/Work/BME2021/MUNKA/theta_workspaces/Logger.log
```

Comment:
These are absolute paths and thus, do not work when imported by another computer. Could you check out if relative paths work here with the project serving as root, e.g., "resources/Logger.log" instead of "D:/2-Programming/Work/BME2021/MUNKA/theta_workspaces/Logger.log"?